--- jupyter_notebook_config.py.orig     2019-10-24 15:27:12.145659367 +0000
+++ jupyter_notebook_config.py  2019-10-24 15:35:29.173654627 +0000
@@ -82,7 +82,7 @@
 #c.NotebookApp.allow_remote_access = False

 ## Whether to allow the user to run the notebook as root.
-#c.NotebookApp.allow_root = False
+c.NotebookApp.allow_root = True

 ## DEPRECATED use base_url
 #c.NotebookApp.base_project_url = '/'
@@ -202,6 +202,7 @@

 ## The IP address the notebook server will listen on.
 #c.NotebookApp.ip = 'localhost'
+c.NotebookApp.ip = '0.0.0.0'

 ## Supply extra arguments that will be passed to Jinja environment.
 #c.NotebookApp.jinja_environment_options = {}
@@ -258,7 +259,7 @@
 #c.NotebookApp.nbserver_extensions = {}

 ## The directory to use for notebooks and kernels.
-#c.NotebookApp.notebook_dir = ''
+c.NotebookApp.notebook_dir = '/workspace/centernet'

 ## Whether to open in a browser after starting. The specific browser used is
 #  platform dependent and determined by the python standard library `webbrowser`
@@ -273,7 +274,7 @@
 #    from notebook.auth import passwd; passwd()
 #
 #  The string should be of the form type:salt:hashed-password.
-#c.NotebookApp.password = ''
+c.NotebookApp.password = ''

 ## Forces users to use a password for the Notebook server. This is useful in a
 #  multi user environment, for instance when everybody in the LAN can access each
@@ -284,7 +285,7 @@
 #c.NotebookApp.password_required = False

 ## The port the notebook server will listen on.
-#c.NotebookApp.port = 8888
+c.NotebookApp.port = 8888

 ## The number of additional ports to try if the specified port is not available.
 #c.NotebookApp.port_retries = 50
@@ -337,7 +338,7 @@
 #
 #  Setting to an empty string disables authentication altogether, which is NOT
 #  RECOMMENDED.
-#c.NotebookApp.token = '<generated>'
+c.NotebookApp.token = ''

 ## Supply overrides for the tornado.web.Application that the Jupyter notebook
 #  uses.
